fastly/terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
This is a lockfile, the same concept as npm's `package-lock.json` but for terraform, it ensures we use the same versions of the fastly provider plugin

Reply:
Nice!